Update README.md for new release (#61)
liquidata-inc/dolt: README.md

## Adding Remotes

Dolt supports remotes in a similar manner to `Git`. Liquidata, the company behind `Dolt`, also created DoltHub, a hosting service for `Dolt` databases. In the following we use DoltHub as an example for setting up a remote.

If you haven't done so already, setting up your default servers will make it easier to add and clone remotes

    dolt config --global --add creds.add_url https://www.dolthub.com/settings/credentials remotes.default_host doltremoteapi.dolthub.com remotes.default_port 443

Next you'll want to make sure you've created the remote at https://www.dolthub.com .  Once created you can add the remote.  As an example, if the repository is created under an organization named "org", with the name "repo" you could add the remote like so:

    dolt remote add origin org/repo

Once the remote is added viewing the remote using the command:

    dolt remote -v

should show something like this:

    origin https://doltremoteapi.dolthub.com/org/repo

If you've created the repository on the web and added a remote, you should be able to push the branch "master" to the remote named "origin" like so

    dolt push origin master

> Once that is succeeded others can clone the repository (assuming you've given them permission.)

Reviewer: ```suggestion
Once that is succeeded others can clone the repository (assuming you've given them permission):
```

    dolt clone org/repo

## Other remotes

> dolt also supports directory, aws, and gcs based remotes:

Reviewer: ```suggestion
Dolt also supports directory, AWS, and GCS based remotes:
```

>   * file - you can use a directory as a remote that can be pushed to, cloned, and purlled from just like any other remote by providing a file uri for the directory

Reviewer: ```suggestion
  * file - you can use a directory as a remote that can be pushed to, cloned, and pulled from just like any other remote by providing a file URI for the directory
```

    dolt remote add <remote> file:///Users/xyz/abs/path/

>   * aws - you can use your aws cloud resources directly (If you are interested in trying this contact the dolt team directly as we are still writing documentation).

Reviewer: ```suggestion
  * AWS - you can use your AWS cloud resources directly (If you are interested in trying this contact the Dolt team directly as we are still writing documentation).
```

    dolt remote add <remote> aws://dynamo-table:s3-bucket/database

>   * gs - you can use a GCS bucket as well.  Setup here should be straight forward.  You'll need to create a gcs bucket, and you'll need to use the gcloud auth login command in order to setup your credentials.

Reviewer: ```suggestion
  * GCS - you can use a GCS bucket as well.  Setup here should be straight forward.  You'll need to create a GCS bucket, and you'll need to use the gcloud auth login command in order to set up your credentials.
```

    dolt remote add <remote> gs://gcs-bucket/database

## Issues

> If you have any issues with `Dolt`, find any bugs, or simply have a question, feel free to file an issue!

Reviewer: ```suggestion
If you have any issues with Dolt, find any bugs, or simply have a question, feel free to file an issue!
```
